jwql/website/apps/jwql/forms.py

```diff
@@ -45,18 +45,38 @@ def view_function(request):
 import datetime
 import glob
 import os
+import inflection
 
 from astropy.time import Time, TimeDelta
 from django import forms
 from django.shortcuts import redirect
 from jwedb.edb_interface import is_valid_mnemonic
 
+# from data_containers import get_thumbnails_all_instruments
 from jwql.database import database_interface as di
-from jwql.utils.constants import ANOMALY_CHOICES, JWST_INSTRUMENT_NAMES_SHORTHAND
+from jwql.utils.constants import ANOMALY_CHOICES, FILTERS_PER_INSTRUMENT
```

Comment:
Importing two things per line here seems a bit arbitrary and may make things difficult if we are trying to measure what/how many things we are importing from a repository-level, so I would suggest just importing one thing per line here. I think this is what PEP8 suggests too.